app/src/main/res/values/dimension.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
I defined a file `dimen.xml` to store dimensions, it might a good idea to merge it into yours instead of keeping the 2 versions